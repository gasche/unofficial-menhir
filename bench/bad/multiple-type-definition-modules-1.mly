%type<int> a
%%

%public a: {}
