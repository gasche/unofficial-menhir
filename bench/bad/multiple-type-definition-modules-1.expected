File "multiple-type-definition-modules-2.mly", line 1, characters 12-13:
Error: There are multiple %type definitions for nonterminal a.
